spec/lib/report_formater/jqplot_formater_spec.rb

```diff
@@ -34,7 +38,7 @@
             ["tak", 1, system_name_4,          359],
           ],
         )
-
+        
```

Comment:
@martinpovolny just noticed that directory name has the same issue.  Someone should rename `report_formater` to `report_formatter` in a separate PR.